changelog.md

```diff
@@ -30,6 +31,7 @@
 
 * Added API for notes, i.e. `file.notes` returns a dictionary of notes.
 * Added option to add a scale bar to by providing a [`lk.ScaleBar()`](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.ScaleBar.html) to plotting or export functions.
+* Added extra method for centroid refinement that shrinks the window to reduce estimation bias.
```

Comment:
this seems a bit oddly worded, at least for me I immediately thought of a method on a class. Maybe instead, we should mention explicitly the added `bias_correction` keyword arg?

Reply:
Good point. I changed it to:
```
* Implemented bias correction for centroid refinement that shrinks the window to reduce estimation bias. This bias correction can be toggled by passing a `bias_correction` argument to [`lk.track_greedy()`](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.track_greedy.html#) and [`lk.refine_tracks_centroid()`](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.refine_tracks_centroid.html).
```